introduction.md

```diff
@@ -1,2 +1,4 @@
 jarios is in the dojo. Can we call this jojo?
 My favorite course is Javascript.
+
+**Git Ninja**
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?